test/files/presentation/callcc-interpreter/Runner.scala

```diff
@@ -1,3 +1,6 @@
 import scala.tools.nsc.interactive.tests._
 
-object Test extends InteractiveTest
\ No newline at end of file
+object Test extends InteractiveTest {
+  // Normalize ordering of LUB
+  override def normalize(s: String) = s.replace("Serializable with Product", "Product with Serializable")
```

Comment:
It turned out to be non-determistic listing order of directories on, e.g. ext4, combined with the way that the `Symbol#id` is used a tie breaker for ordering elements of the base type sequence that have the same base type depth.

Fixed in https://github.com/scala/scala/pull/5263/commits/c7ec84f3b99b587c8dc60b4eda61ab2a939ab68e